### DISCUSSION TITLE
Swagger for ocelot API's get 404 error while switching API List

### DISCUSSION THREAD

**Discussion by vishnusurendran-os** [Burgyn/MMLib.SwaggerForOcelot#199] [Q&A]
when the api get 404 error when switching api list 
here is the sample code https://github.com/vishnusurendran-os/SwaggerForOcelot

**Comment by Burgyn** [comment]
Hi,

could you please be a little more specific? What exactly does `404` return to you? Because when I run your example, the documentation will appear for both Test1 and Test2.

**Reply by vishnusurendran-os** [reply]
The 401 error get in test 2 API. that switch the api from Test 1 to Test2

**Reply by Burgyn** [reply]
I am after appendix surgery.  I'll look into it when I get back from the hospital.

**Reply by Burgyn** [reply]
Hi @vishnusurendran-os,

this question should be related at the Ocelot project and not here. Because you have a problem with the Ocelot settings. 
You are missing `.Wait()` method in `Startup.cs` 
```csharp
app.UseOcelot().Wait();
```
